kotlin/src/test/kotlin/com/linecorp/armeria/internal/server/annotation/DataClassDefaultNameTypeInfoProviderTest.kt

```diff
@@ -82,7 +87,10 @@ class DataClassDefaultNameTypeInfoProviderTest {
         @Description(value = "default value description")
         val defaultValue: String = "Hello",
         @Description(value = "default value 2 description")
-        val defaultValue2: String = "Hello2"
+        val defaultValue2: String = "Hello2",
+        @JsonProperty("renamed")
+        @Description("renamed description")
+        val name: String?
```

Comment:
Should we also add a test for a non-null value?

Reply:
Thanks for the review. I will add a test for that.